Application/EdFi.Ods.Api/Security/Authorization/AuthorizationSegmentSqlProviderBase.cs

```diff
@@ -45,100 +46,83 @@ public AuthorizationSegmentSqlProviderBase(IAuthorizationViewsProvider authoriza
         {
             var segmentStatements = new List<string>();
             var parameters = new List<DbParameter>();
+            var unsupportedAuthorizationViews = new List<string>();
+            var edOrgTypes = new List<string>();
 
             foreach (var authorizationSegment in authorizationSegments)
             {
-                // Within each claim segment, group the values by ed org type, and combine with "OR"
-                var claimEndpointsGroupedByName = authorizationSegment.ClaimsEndpoints
-                    .GroupBy(ep => ep.Name)
-                    .Select(g => g)
-                    .ToList();
+                string claimEndpointName = authorizationSegment.ClaimsEndpoints.FirstOrDefault().Name;
 
-                var segmentExpressions = new List<string>();
+                if (EducationOrganizationEntitySpecification.IsEducationOrganizationIdentifier(claimEndpointName))
+                {
+                    claimEndpointName = "EducationOrganizationId";
+                }
```

Comment:
Let's go ahead and close a possible exceptional condition that we're not supporting right now by adding an `else` clause and a `throw new NotSupportedException($"Claim endpoint name of '{claimEndpointName}' is not yet supported for authorization.");`.